github/commands/create_fork.py

```diff
@@ -0,0 +1,42 @@
+import sublime
+from sublime_plugin import WindowCommand
+
+from ...common import util
+from ...core.git_command import GitCommand
+from ...core.ui_mixins.quick_panel import PanelCommandMixin
+from .. import github, git_mixins
+
+START_CREATE_MESSAGE = "Creating fork of {repo} ..."
+END_CREATE_MESSAGE = "Fork created successfully."
+
+
+__all__ = ['GsCreateForkCommand']
+
+
+class GsCreateForkCommand(PanelCommandMixin, WindowCommand, GitCommand,
+                          git_mixins.GithubRemotesMixin):
+    """
+    Get list of repos on GitHub associated with the active repo.  Display, and when
+    selected, add selection as git remote.
+    """
+    def run(self):
+        sublime.set_timeout_async(self.run_async, 0)
+
+    def run_async(self):
+        base_remote = github.parse_remote(self.get_integrated_remote_url())
+        sublime.status_message(START_CREATE_MESSAGE.format(repo=base_remote))
+        result = github.create_fork(base_remote)
+        sublime.status_message(END_CREATE_MESSAGE)
+        util.debug.add_to_log(("github: fork result:\n{}".format(result)))
+        self.window.show_quick_panel(
```

Comment:
I quite like this kind of prompt, way better than `sublime.ok_cancel_dialog`. I think we should create our version of `ok_cancel_prompt` based on `quick_panel`. But it is a story of another time.